save-plugins/fix-plugin/src/commonMain/kotlin/com/saveourtool/save/plugins/fix/FixPlugin.kt

```diff
@@ -247,6 +247,17 @@ class FixPlugin(
         )
     }
 
+    private fun isComparingTestAndCopy(test: Path, testCopy: Path): Boolean {
+        val testCopyPath = testCopy.relativeTo(FileSystem.SYSTEM_TEMPORARY_DIRECTORY)
```

Comment:
Could you also write a comments, why do we calculate paths in this way?
Some words like: `testCopyPath` stored in `tmpDir`, holdind the whole hierarchy of original file
while `test` file comes to us with path , starting from `testRootPath`, so we compare them in such way